src/helpers/config.ts

```diff
@@ -2,6 +2,7 @@ import { NetworkInfo, NetworkType } from "@/interfaces/network";
 
 export const UP_CONNECTED_ADDRESS = "up:connected-address";
 
+export const DEFAULT_IPFS_URL = "https://2eff.lukso.dev/ipfs/";
```

Comment:
We already have ipfs definied in `NETWORKS`

Reply:
Oh! didn't know